benchmark_results.md

```diff
@@ -0,0 +1,96 @@
+# Benchmark Results
+
+Following numbers were gathered targeting the [process-quarkus-example](https://github.com/kiegroup/kogito-examples/tree/stable/process-quarkus-example) on different compute environments.
+
+Mainly 2 options were tested:
+1. Kogito application deployed in a Virtual machine
+2. Kogito application deployed in a Red Hat Openshift cluster.
+
+
+#### VM Details: 🖥️
+TBD
+
+#### Config Details: ⚒️
+
+- Java version: openjdk 11.0.12 2021-07-20 LTS
+- Persistence: Mongo DB v4.4.9
+- Kogito version: 1.11.0.Final
+- Concurrence Strategy: CONST_CONCURRENT_USERS
+
+## VM deployment
+
+|   🧰 Test Engine	|   Business Process Name ➙ Config Name	| Persistence ➙ Config Name | Replicas 	|  👥 Users 	|  ⏰ Duration 	|  🔀 Request count 	|  ⚠️ Ok/KO count 	| ⚡ mean response time | 🛎️ mean requests/sec 	|
+|---	|---	|:-----:	|:---:|    ---:|  :---: | ---:| :---:| ---:| ---:|
+|  Gatling	|  emptyProcess ➙ default |  ❌ No persistence	|  1 	|  1 	|   5 min	|  30688 	|  28232/2456 	|  2 ms 	|   503.08/sec	|
+|  Gatling 	|  emptyProcess ➙ default |  ❌ No persistence	|  1 	|  4 	|   5 min	|  54197 	|  28232/25965 	|  4 ms 	|   903.28/sec	|
+|  Gatling 	|  emptyProcess ➙ default |  ❌ No persistence	|  1 	|  8 	|   5 min	|  60516 	|  30875/29641 	|  5 ms 	|   992.06/sec	|
+|  Gatling 	|  emptyProcess ➙ default |  ❌ No persistence	|  1 	|  20 	|   5 min	|  65462 	|  33255/32207 	|  10 ms 	|   1073.14/sec	|
+|  Gatling 	|  emptyProcess ➙ default |  ❌ No persistence	|  1 	|  40 	|   5 min	|  318394 	|  148050/170344 	|  19 ms 	|  1057.78/sec 	|
+|  Gatling 	|  emptyProcess ➙ default |  ❌ No persistence	|  1 	|  60 	|   5 min	|  334250 	|  147883/186367 	|  28 ms 	|  1110.46/sec 	|
+|  Gatling 	|  notPersistedProcess ➙ default |  ❌ No persistence	|  1 	|  1 	|   5 min	|  150819 	|  131159/19660 	|  2 ms 	|  501.06/sec 	|
+|  Gatling 	|  notPersistedProcess ➙ default |  ❌ No persistence	|  1 	|  4 	|   5 min	|  173586 	|  144145/29441 	|  2 ms 	|  576.69/sec 	|
+|  Gatling 	|  notPersistedProcess ➙ default |  ❌ No persistence	|  1 	|  8 	|   5 min	|  178186 	|  146601/31585 	|  3 ms 	|  591.98/sec 	|
+|  Gatling 	|  notPersistedProcess ➙ default |  ❌ No persistence	|  1 	|  20 	|   5 min	|  181766 	|  141168/40598 	|  12 ms 	|  603.87/sec 	|
+|  Gatling 	|  notPersistedProcess ➙ default |  ❌ No persistence	|  1 	|  40 	|   5 min	|  187024 	|  153349/33675 	|  27 ms 	|  621.34/sec 	|
+|  Gatling 	|  notPersistedProcess ➙ default |  ❌ No persistence	|  1 	|  60 	|   5 min	|  194124 	|  150145/43979 	|  43 ms 	|  644.93/sec 	|
+|  Java-Quarkus 	|  simple ➙ default |  ✅ Mongo DB ➙ clean-before	|  1 	|  1 	|   2 min	|  10777 	|  10777/0 	|  10 ms 	|  89/sec 	|
```

Comment:
Can you add quick description of what "clean-before" and "id-index" mean, from a procedure perspective?